css/dashboard.css

```diff
@@ -6,33 +6,41 @@
   height: var(--top-nav-height);
   justify-content: center;
 }
+
+h2{
+  font-size: 30px;
```

Comment:
why do we need to change size for h2? 
As general rule of them, don't change the css style of generic element such as h1, h2, button. Instead you can make a class for the h2 then put your font-size there